src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

```diff
@@ -29,42 +33,48 @@
 /**
  * @author VeeWee <toonverwerft@gmail.com>
  */
-final class FullyQualifiedStrictTypesFixer extends AbstractFixer
+final class FullyQualifiedStrictTypesFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
     /**
      * {@inheritdoc}
      */
     public function getDefinition(): FixerDefinitionInterface
     {
         return new FixerDefinition(
-            'Transforms imported FQCN parameters and return types in function arguments to short version.',
+            'Transforms FQCN parameters and return types in function arguments, caught exception `classes`, `extend` and `implements` of `classes` and `interfaces` to short version.',
```

Comment:
I'm struggling to understand what this fixer does. Maybe something like this would be easier to understand?

```suggestion
            'Removes the leading part of fully qualified symbol references if a given symbol is imported or belongs to the current namespace.',
```

Also, what does “strict types” in the fixer's name mean?

Reply:
That one got introduces in 2017.
It has been so long that I honestly don't remember anymore.

I think the name origines from the return types and strict mode which got introduced around that time in PHP70 if I am not mistaking.
Looking back at the name, it might be better to drop the strict and have named it something like `FullyQualifiedTypesFixer`.  Not sure it is worth changing though.

You can see the full comments history of that PR here:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3276

Reply:
Thank you for sharing your insight, I couldn't find that PR. Apparently “strict return types” used to mean any return types at all at that time: `The strict return type : operator is only available from PHP 7.0.`.

Back in 2017, I was busy adding more inline PHP to my few-thousand-LOC `index.php` files, so didn't know about that :)